Source/DafnyCore/AST/Statements/Verification/HideRevealStmt.cs

```diff
@@ -96,7 +96,7 @@ public void Resolve(PreTypeResolver resolver, ResolutionContext resolutionContex
           if (effectiveExpr is NameSegment) {
             resolver.ResolveNameSegment((NameSegment)effectiveExpr, true, null, resolutionContext, true);
           } else {
-            resolver.ResolveDotSuffix((ExprDotName)effectiveExpr, true, null, resolutionContext, true);
+            resolver.ResolveDotSuffix((ExprDotName)effectiveExpr, true, true, null, resolutionContext, true);
```

Comment:
These boolean flags are hard to review on GitHub, Non blocking for this one, but I would encourage in the future that we switch to enums when it's not obvious what a boolean is about. For example, `StaticReferenceToInstance.Allowed`.